### ISSUE TITLE
Capitalize "text alternative" for consistency in Section 3.

### ISSUE THREAD

**Issue by skynavga** [w3c/imsc#160]

**Comment by palemieux** [comment]
"text alternative" is not capitalized in the WCAG glossary at https://www.w3.org/TR/WCAG20/#text-altdef .

**Comment by skynavga** [comment]
That's ok, we should be consistent in our own use of vocabulary, we don't
have to keep the uncapitalized form from WCAG.
